res/menu/gallery_drawer.xml

```diff
@@ -3,14 +3,26 @@
     xmlns:mupen64="http://schemas.android.com/apk/res-auto" >
 
     <item
+        android:id="@+id/menuItem_library"
+        android:icon="@drawable/ic_controller"
+        android:title="@string/categoryLibrary_title"/>
+
+    <item
         android:id="@+id/menuItem_settings"
         android:icon="@drawable/ic_settings"
-        android:title="@string/menuItem_settings">
+        android:title="@string/menuItem_settings"/>
+
+    <item
```

Comment:
Language below the Settings should be good enough since it has the Map icon, and Settings will be accessed much more often.